zhaquirks/sinope/light.py

```diff
@@ -237,199 +258,262 @@ def _get_command_from_action(
 class LightManufacturerCluster(EventableCluster, SinopeTechnologiesManufacturerCluster):
     """LightManufacturerCluster: fire events corresponding to press type."""
 
+    def _update_attribute(self, attrid, value):
+        if attrid == self.AttributeDefs.current_summation_delivered.id:
+            value = value / 100
+        super()._update_attribute(attrid, value)
+
+
+(
+    # <SimpleDescriptor endpoint=1 profile=260 device_type=259
+    # device_version=0 input_clusters=[0, 2, 3, 4, 5, 6, 1794, 2821, 65281]
+    # output_clusters=[3, 4, 25]>
+    QuirkBuilder(SINOPE, "SW2500ZB")
+    .applies_to(SINOPE, "SW2500ZB-G2")
+    .replaces_endpoint(1, device_type=zha_p.DeviceType.ON_OFF_LIGHT)
+    .adds(Basic, endpoint_id=1)
+    .adds(Identify, endpoint_id=1)
+    .adds(Groups, endpoint_id=1)
+    .adds(Scenes, endpoint_id=1)
+    .adds(OnOff, endpoint_id=1)
+    .adds(Metering, endpoint_id=1)
+    .adds(Diagnostic, endpoint_id=1)
```

Comment:
You shouldn't be doing this in v2 quirks. Only add clusters if they're missing from the device signature by default.
This won't do anything, since they're most likely all already exposed by the device itself.

Reply:
The issue with `replaces_endpoint` should have been fixed with https://github.com/zigpy/zigpy/pull/1602. Can you confirm if you can now remove these `adds` lines?